<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materials_panel</class>
 <widget class="QWidget" name="materials_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>271</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0" colspan="4">
      <widget class="QPushButton" name="show_overlay_manager">
       <property name="text">
        <string>Overlay Manager</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="max_tth_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Max 2θ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="limit_active">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Limit the allowed HKLs with the max 2θ or the minimum d spacing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Limit Active</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QComboBox" name="materials_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>CeO2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LaB6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ruby</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Silicon</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QTabWidget" name="tab_widget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="general_tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="2">
          <widget class="ScientificDoubleSpinBox" name="min_d_spacing">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum d spacing for the HKLs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> Å</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="show_overlays">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display overlays in the canvas for each material&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Show Overlays</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="max_tth_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Max 2θ:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="limit_active">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Limit the allowed HKLs with the max 2θ or the minimum d spacing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Limit Active</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QPushButton" name="show_overlay_manager">
           <property name="text">
            <string>Overlay Manager</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QPushButton" name="show_reflections_table">
           <property name="text">
            <string>Reflections Table</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="min_d_spacing_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Min d-spacing:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="ScientificDoubleSpinBox" name="max_tth">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The max 2θ for the HKLs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0" colspan="4">
          <layout class="QVBoxLayout" name="material_editor_layout"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="structure_tab">
        <attribute name="title">
         <string>Structure</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="material_structure_editor_layout"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="properties_tab">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="material_properties_editor_layout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="ScientificDoubleSpinBox" name="min_d_spacing_displayed">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum d spacing for the HKLs to be displayed. This is applied to the entries in the &amp;quot;Materials Table&amp;quot; as well as to any powder overlays on display. It does not apply to Laue overlays, however.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> Å</string>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QPushButton" name="show_materials_table">
       <property name="text">
        <string>Materials Table</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="materials_tool_button">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to add, modify, or delete material&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton::menu-indicator { image: none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="ScientificDoubleSpinBox" name="max_tth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The max 2θ for the HKLs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="min_d_spacing_displayed_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum d spacing for the HKLs to be displayed. This is applied to the entries in the &amp;quot;Materials Table&amp;quot; as well as to any powder overlays on display. It does not apply to Laue overlays, however.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Min d-spacing (displayed):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="show_overlays">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display overlays in the canvas for each material&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Show Overlays</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="min_d_spacing_label">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum d-spacing for the HKLs to be generated by hexrd.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Min d-spacing:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="ScientificDoubleSpinBox" name="min_d_spacing">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum d-spacing for the HKLs to be generated by hexrd.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> Å</string>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>materials_combo</tabstop>
  <tabstop>materials_tool_button</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>show_reflections_table</tabstop>
  <tabstop>show_overlay_manager</tabstop>
  <tabstop>show_overlays</tabstop>
  <tabstop>min_d_spacing</tabstop>
  <tabstop>limit_active</tabstop>
  <tabstop>max_tth</tabstop>
  <tabstop>min_d_spacing_displayed</tabstop>
  <tabstop>tab_widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
